README.md

```diff
@@ -1 +1,24 @@
-# hello-rails-react-
\ No newline at end of file
+# README
+
+This README would normally document whatever steps are necessary to get the
+application up and running.
+
+Things you may want to cover:
+
+* Ruby version
+
+* System dependencies
+
+* Configuration
+
+* Database creation
+
+* Database initialization
+
+* How to run the test suite
```

Comment:
## OPTIONAL 
* I strongly suggest you update your readme file to contain useful information on how to properly set up and run the project [Use the microverse readme template](https://github.com/microverseinc/readme-template)